src/app/pages/questions/components/question/question.component.ts

```diff
@@ -59,6 +61,13 @@ export class QuestionComponent implements OnInit, OnChanges {
   ])
   MATRIX_INPUT_SET: Set<QuestionType> = new Set([QuestionType.matrix_radio])
 
+  // Input set where height is set to auto
+  AUTO_HEIGHT_INPUT_SET: Set<QuestionType> = new Set([
```

Comment:
why not for all types?

Reply:
Because the other input type heights (like info screen input type height) are calculated based on device specs, so for these there's no need to calculate. We just set them as auto.